rwslib/builders/admindata.py

```diff
@@ -0,0 +1,391 @@
+# -*- coding: utf-8 -*-
+
+from rwslib.builders.common import ODMElement, dt_to_iso8601
+from rwslib.builders.clinicaldata import LocationRef
+from rwslib.builders.constants import LocationType, UserType
+from rwslib.builders.modm import LastUpdateMixin
+
+
+class AdminData(ODMElement):
+    """
+    Administrative information about users, locations, and electronic signatures.
+    """
+    def __init__(self, study_oid=None):
+        """
+        :param str study_oid: OID pointing to the StudyDef
+        """
+        super(AdminData, self).__init__()
+        self.study_oid = study_oid
+        self.users = []
+        self.locations = []
+        # SignatureDef
+
+    def build(self, builder):
+        """Build XML by appending to builder"""
+        params = {}
+        if self.study_oid:
+            params.update(dict(StudyOID=self.study_oid))
+        builder.start("AdminData", params)
+        for user in self.users:
+            user.build(builder)
+        for location in self.locations:
+            location.build(builder)
+
+        builder.end("AdminData")
+
+    def __lshift__(self, other):
```

Comment:
Good example `odm << clinical_data << ...<< itemgroup` that justifies the override, I'm on board with that now ;)